File: SoalB/PHP/RAM.php
<?php

class RAM
{
    /* atribut */
    private $priceRAM, $capacityRAM;

    /* constructor */
    function __construct($priceRAM, $capacityRAM)
    {
        $this->priceRAM = $priceRAM;
        $this->capacityRAM = $capacityRAM;
    }

    /* membuat get set method untuk harga RAM */
    function setPriceRAM($priceRAM)
    {
        $this->priceRAM = $priceRAM;
    }
    function getPriceRAM()
    {
        return $this->priceRAM;
    }

    /* Membuat get set method kapasitas RAM */
    function setCapacityRAM($capacityRAM)
    {
        $this->capacityRAM = $capacityRAM;
    }
    function getCapacityRAM()
    {
        return $this->capacityRAM;
    }

    /* menampilkan hasil data RAM*/
    function printRAM()
    {
        echo "RAM Price(IDR)        : " . $this->getPriceRAM() . "<br/>";
        echo "RAM Capacity(GB)      : " . $this->getCapacityRAM() . "<br/>";
        echo "******************************" . "<br/>";
    }

    /*destructor */
    function __destruct()
    {
    }
}

File: SoalB/Java/RAM.java
public class RAM {
    /* atribut */
    private int priceRAM;
    private int capacityRAM;

    /* constructor */
    public RAM(int priceRAM, int capacityRAM) {
        this.priceRAM = priceRAM;
        this.capacityRAM = capacityRAM;
    }

    public RAM() {

    }

    /* membuat get set methods harga RAM */
    public void setPriceRAM(int priceRAM) {
        this.priceRAM = priceRAM;
    }

    public int getPriceRAM() {
        return priceRAM;
    }

    /* membuat get set methods kapasitas RAM */
    public void setCapacityRAM(int capacityRAM) {
        this.capacityRAM = capacityRAM;
    }

    public int getCapacityRAM() {
        return capacityRAM;
    }

    /* menampilkan data RAM */
    public void printRAM() {
        System.out.println("RAM Price(IDR)      : " + this.getPriceRAM());
        System.out.println("RAM Capacity(GB)    : " + this.getCapacityRAM());
        System.out.println("######################################");
    }
}
